in-adapter-api/src/main/java/gohigher/support/enumconvertor/ProcessTypeConvertor.java

```diff
@@ -0,0 +1,13 @@
+package gohigher.support.enumconvertor;
+
+import org.springframework.core.convert.converter.Converter;
+
+import gohigher.common.ProcessType;
+
+public class ProcessTypeConvertor implements Converter<String, ProcessType> {
+
+	@Override
+	public ProcessType convert(String value) {
+			return ProcessType.from(value);
+	}
+}
```

Comment:
```suggestion
public class ProcessTypeConvertor implements Converter<String, ProcessType> {

	@Override
	public ProcessType convert(String value) {
		return ProcessType.from(value);
	}
}
```
오홍,, 처음 알았네요 배워갑니다,,,,

Reply:
저도 이번에 알았습니다.. 허허